<launch>
        <node pkg="boot_servo_demo" name="servo_manager" type="servo_demo.py"
        	  required='true' output='screen'>
        	<param name='boot_root' 
        		   value='/data/work/1303-youbot-videos/20-grace-workspace/out/boot-root'/> 
        	<param name='config_dir' 
        		   value='/data/work/scm/boot12env/src/bvapps/1304-youbot/'/> 
			<param name='id_agent' value='exp08_bdser1'/>
			<param name='id_robot' value='exp05_uA_xy'/>
			<param name='id_ros_robot_adapter' value='exp05_uA_xy'/>
			
			<param name='error_threshold' value='0.01'/>
			
			<param name='K' value='[1,1]'/>
			
        </node>
</launch>